src/main/java/org/jabref/migrations/PreferencesMigrations.java

```diff
@@ -328,6 +328,7 @@ protected static void upgradePreviewStyle(JabRefPreferences prefs) {
         String currentPreviewStyle = prefs.get(JabRefPreferences.PREVIEW_STYLE);
         String migratedStyle = currentPreviewStyle.replace("\\begin{review}<BR><BR><b>Review: </b> \\format[HTMLChars]{\\review} \\end{review}", "\\begin{comment}<BR><BR><b>Comment: </b> \\format[HTMLChars]{\\comment} \\end{comment}")
```

Comment:
Reading that, I totally forgot about the "review" field in https://github.com/JabRef/jabref/pull/10896.

Could you add `MARKDOWN` to the `review` field?

Reply:
> Could you add `MARKDOWN` to the `review` field?

Sure